src/celeritas/Types.hh

```diff
@@ -38,14 +38,13 @@ using UniqueEventId = OpaqueId<struct Event_, std::uint64_t>;
 using IsotopeId = OpaqueId<struct IsotopeRecord>;
 
 //! Opaque index of a material modified by physics options
-// TODO: rename to PhysMatId; equivalent to "material cuts couple"
-using MaterialId = OpaqueId<struct Material_>;
+using PhysMatId = OpaqueId<struct PhysicsMaterial_>;
```

Comment:
```suggestion
using PhysCutsMatId = OpaqueId<struct PhysicsCutsMaterial_>;
```
I am leaning toward `Cuts` being an important part of the name/concept.

Reply:
I don't think `Cuts` is useful at all except to people intimately acquainted with geant4. The key difference between two "physics materials" that have the same "geometry material" is that the physics treatment is different in the two. That could be different secondary production cutoff, or a different physics list.